plugin/trino-phoenix5/src/main/java/io/trino/plugin/phoenix5/PhoenixConfig.java

```diff
@@ -52,4 +58,19 @@ public PhoenixConfig setResourceConfigFiles(String files)
         this.resourceConfigFiles = Splitter.on(',').trimResults().omitEmptyStrings().splitToList(files);
         return this;
     }
+
+    @Min(1)
+    @Max(MAX_ALLOWED_SCANS_PER_SPLIT)
+    public int getMaxScansPerSplit()
+    {
+        return maxScansPerSplit;
+    }
+
+    @Config("phoenix.max-scans-per-split")
+    @ConfigDescription("The maximum number of HBase scans that Phoenix will allow per split.")
```

Comment:
The description feels incorrect since Phoenix doesn't have any control I'm guessing.

Maybe change to `Maximum number of HBase scans that can be performed in a single split.`

Reply:
Agreed.